src/formBuilder/formBuilder.jsx

```diff
@@ -80,7 +80,12 @@ const FormBuilder = props => {
   };
 
   const buildABIOptions = (abiString, serialTag = false) => {
-    if (!abiString || typeof abiString !== 'string') return;
+    if (
+      !abiString ||
+      typeof abiString !== 'string' ||
+      abiString.slice(0, 2) === '0x'
```

Comment:
not necessary to implement, but we also have the validate.address() utilities for stuff like this. Might be helpful.

Reply:
Might be good to have a validate.hex() if such a thing is possible.

Reply:
there are `web3.utils.isHexStrict` or `ethers.utils.isHexString`